marlowe-tutorial/doc/introducing-marlowe.adoc

```diff
@@ -109,8 +109,6 @@ This tutorial covers the latest version of Marlowe, while all but the
 first of these links covers Marlowe 1.3, which is very close to Marlowe
 3.0 in conception but which differs in a number of detailed ways. 
 
-* The Marlowe github repository:
-https://github.com/input-output-hk/marlowe[link]
-* The Marlowe paper: https://iohk.io/research/papers/#2WHKDRA8[link]
-* Marlowe video from PlutusFest, December 2018:
-https://www.youtube.com/watch?v=rSpFOADHLqw[link]
+* https://github.com/input-output-hk/marlowe[The Marlowe github repository]
+* https://api.zotero.org/groups/478201/items/L65KWDLX/file/view?key=Qcjdk4erSuUZ8jvAah59Asef[The Marlowe paper]
```

Comment:
I mean the one in a conference - didn't the marlowe paper get published? I don't trust this link either!

Reply:
do you have a link, I have no idea I was just given this

Reply:
Hm, some googling suggests it's paywalled. Disappointing.